src/Search.js

```diff
@@ -17,7 +18,8 @@ const Search = () => {
                 className="form-control"
                 placeholder="Customer name"
               />
-              <button className="btn btn-primary">Search</button>
+              {/* <button className="btn btn-primary">Search</button> */}
+              <SearchButton />
```

Comment:
this is good, you've extracted the button code into a new SearchButton component and imported it, well done